server.py

```diff
@@ -29,12 +46,43 @@ def bad_request(error):
     app.logger.info(message)
     return jsonify(status=400, error='Bad Request', message=message), 400
 
+######################################################################
+#  Index
+######################################################################
+@app.route("/")
+def index():
+    return jsonify(name='Wishlist REST API Service',
+                   version='1.0',
+                   docs=request.base_url + 'apidocs/index.html'), status.HTTP_200_OK
```

Comment:
does this load when I open `/` or do I open `apidocs`? Just confused about the syntax

Reply:
You open /apidoc/index.html. The / indicates which link for api docs